File: Voronezh/Bars.Gkh.Regions.Voronezh/Entities/RosRegExtract/RosRegExtractOrg.cs
using Bars.B4.DataAccess;
using Bars.Gkh.Entities;

namespace Bars.Gkh.Regions.Voronezh.Entities
{
    public class RosRegExtractOrg : PersistentObject
    {        
        //Organization
        public virtual string Org_Code_SP { get; set; }
        public virtual string Org_Content { get; set; }
        public virtual string Org_Code_OPF { get; set; }
        public virtual string Org_Name { get; set; }
        public virtual string Org_Inn { get; set; }
        public virtual string Org_Code_OGRN { get; set; }
        public virtual string Org_RegDate { get; set; }
        public virtual string Org_AgencyRegistration { get; set; }
        public virtual string Org_Code_CPP { get; set; }
        //Organization->Location
        public virtual string Org_Loc_ID_Address { get; set; }
        public virtual string Org_Loc_Content { get; set; }
        public virtual string Org_Loc_RegionCode { get; set; }
        public virtual string Org_Loc_RegionName { get; set; }
        public virtual string Org_Loc_Code_OKATO { get; set; }
        public virtual string Org_Loc_Code_KLADR { get; set; }
        public virtual string Org_Loc_CityName { get; set; }
        public virtual string Org_Loc_StreetName { get; set; }
        public virtual string Org_Loc_Level1Name { get; set; }
        //Organization->FactLocation
        public virtual string Org_FLoc_ID_Address { get; set; }
        public virtual string Org_FLoc_Content { get; set; }
        public virtual string Org_FLoc_RegionCode { get; set; }
        public virtual string Org_FLoc_RegionName { get; set; }
        public virtual string Org_FLoc_Code_OKATO { get; set; }
        public virtual string Org_FLoc_Code_KLADR { get; set; }
        public virtual string Org_FLoc_CityName { get; set; }
        public virtual string Org_FLoc_StreetName { get; set; }
        public virtual string Org_FLoc_Level1Name { get; set; }
    }
}

File: Voronezh/Bars.Gkh.Regions.Voronezh/libs/B4/model/RosRegExtractOrg.js
Ext.define('B4.model.RosRegExtractOrg', {
    extend: 'B4.base.Model',
    idProperty: 'Id',

    proxy: {
        type: 'b4proxy',
        controllerName: 'RosRegExtractOrg'
    },
    fields: [
        { name: 'Id' },
        { name: 'Org_Code_SP' },
        { name: 'Org_Content' },
        { name: 'Org_Code_OPF' },
        { name: 'Org_Name' },
        { name: 'Org_Inn' },
        { name: 'Org_Code_OGRN' },
        { name: 'Org_RegDate' },
        { name: 'Org_AgencyRegistration' },
        { name: 'Org_Code_CPP' },
        { name: 'Org_AreaUnit' },
        { name: 'Org_Loc_ID_Address' },
        { name: 'Org_Loc_Content' },
        { name: 'Org_Loc_RegionCode' },
        { name: 'Org_Loc_RegionName' },
        { name: 'Org_Loc_Code_OKATO' },
        { name: 'Org_Loc_Code_KLADR' },
        { name: 'Org_Loc_CityName' },
        { name: 'Org_Loc_StreetName' },
        { name: 'Org_Loc_Level1Name' },
        { name: 'Org_FLoc_ID_Address' },
        { name: 'Org_FLoc_Content' },
        { name: 'Org_FLoc_RegionCode' },
        { name: 'Org_FLoc_RegionName' },
        { name: 'Org_FLoc_Code_OKATO' },
        { name: 'Org_FLoc_Code_KLADR' },
        { name: 'Org_FLoc_CityName' },
        { name: 'Org_FLoc_StreetName' },
        { name: 'Org_FLoc_Level1Name' }
    ]
});